error: local variable path is accessed from within inner class; needs to be declared final ((IonicWebViewEngine)webView.getEngine()).setServerBasePath(path);

When I try to build to Android, I get the following error:

.../platforms/android/src/com/ionicframework/cordova/webview/IonicWebView.java:16: error: 
local variable path is accessed from within inner class; needs to be declared final
((IonicWebViewEngine)webView.getEngine()).setServerBasePath(path);

Looks like it's related to this PR https://github.com/ionic-team/cordova-plugin-ionic-webview/pull/115

My setup looks like this:

cli packages: (node_modules)

    @ionic/cli-utils  : 1.19.2
    ionic (Ionic CLI) : 3.20.0

global packages:

    cordova (Cordova CLI) : 7.1.0 
    Gulp CLI              : CLI version 3.9.1 Local version 3.9.1

local packages:

    Cordova Platforms : android 6.4.0
    Ionic Framework   : ionic1 1.3.5

System:

    ios-deploy : 1.9.2 
    Node       : v8.9.4
    npm        : 2.15.12 
    OS         : macOS High Sierra
    Xcode      : Xcode 9.4 Build version 9F1027a 

Environment Variables:

    ANDROID_HOME : /Library/Android/sdk

Misc:

    backend : pro

--- a/src/android/com/ionicframework/cordova/webview/IonicWebView.java
+++ b/src/android/com/ionicframework/cordova/webview/IonicWebView.java
@@ -10,7 +10,7 @@ public class IonicWebView extends CordovaPlugin  {
   public boolean execute(String action, JSONArray args, CallbackContext callbackContext) throws JSONException {
   
     if (action.equals("setServerBasePath")) {
-      String path = args.getString(0);
+      final String path = args.getString(0);
       cordova.getActivity().runOnUiThread(new Runnable() {
         public void run() {
           ((IonicWebViewEngine)webView.getEngine()).setServerBasePath(path);

